oeq-ts-rest-api/test/TestConfig.ts

```diff
@@ -0,0 +1,3 @@
+export const API_PATH = 'http://localhost:8080/rest/api';
+export const USERNAME = 'TLE_ADMINISTRATOR';
+export const PASSWORD = 'abc';
```

Comment:
Nice to have, a newline at end of file
```suggestion
export const PASSWORD = 'abc';

```